app/controllers/users_controller.rb

```diff
@@ -1,2 +1,7 @@
 class UsersController < ApplicationController
+
+  def show
+    @games = Game.all
```

Comment:
Unfortunately we don't. Without it here, we get the following error:
```
NoMethodError in Users#show
Showing /vagrant/src/metchess/app/views/users/show.html.erb where line #9 raised:

undefined method `each' for nil:NilClass
Extracted source (around line #9):
     <strong>As White Player:</strong>
     <br />
     <% @games.each do |game| %>
```
I've been trying to figure out if there's something wrong in the associations that is causing this, but adding it here fixed it, so I went with it. :)